packages/shared/src/components/comments/CommentContainer.tsx

```diff
@@ -145,8 +150,23 @@ export default function CommentContainer({
                 @{comment.author.username}
               </TruncateText>
             </ProfileLink>
-            <div className="mx-2 h-0.5 w-0.5 bg-text-quaternary" />
+            <Separator />
             <CommentPublishDate comment={comment} />
+            {topReader && (
+              <>
+                <Separator />
+                <SimpleTooltip
```

Comment:
Why do we tooltip here and not on the other one?
Maybe we can abstract into common component rather?

Reply:
Mostly because `UserShortInfo` is used in the user lists, where that row already has a tooltip, so adding a second tooltip doesn't really work well 😬

Reply:
Yeah, I can do that :)